vue-client/src/components/search/facet-group.vue

```diff
@@ -167,11 +181,24 @@ export default {
     hasScroll() {
       return !this.revealLevels[this.currentLevel] && this.isExpanded;
     },
+    isChangeFacetGroup() {
+      return (this.group.dimension === 'category.@id' || this.group.dimension === 'concerning.@reverse.itemOf.heldBy.@id') && this.isChangeView;
+    },
```

Comment:
Is this check necessary? Which facets are we filtering out?
We want `@type` here as well?

Reply:
Yes, this was introduced to explicitly hide the @type facets, but if we want to show that I can remove this.